### DISCUSSION TITLE
multi arch testing

### DISCUSSION THREAD

**Discussion by unixfox** [searx/searx#2491] [Development]
@return42 Let's move here

> Yes, this is what I am looking for: _bug tracking on local PC (i86)._
> 
> Quemu might be one alternative? .. I have to read some articles ..

Well qemu is just a frontend for KVM, so it's not an alternative but more like a tool for using KVM.

With qemu it's possible to run binaries for other architectures with qemu-arm-static like in this example: https://gist.github.com/luk6xff/9f8d2520530a823944355e59343eadc1

**Comment by return42** [comment]
Thanks for the link .. here is what I will try to test: https://askubuntu.com/a/1223581  .. also using qemu-arm-static  and [binfmt_misc](https://en.wikipedia.org/wiki/Binfmt_misc) .. the charm is, that (I hope) I can integrate it in searx's LXC suite.
